FERMAT-WHITE-PAPER.md

```diff
@@ -77,7 +77,7 @@ Each node maintains the Fermat Blockchain: a public record of all _coinbase_ tra
 
 ### Fermat Clients
 
-Fermat clients run the Fermat Framework, which in turn run the Fermat Components (libraries, add-ons, plug-ins, GUIs, etc.). Clients adhere to the Fermat Client Protocol.
+Fermat clients run the Fermat Framework, which in return run the Fermat Components (libraries, add-ons, plug-ins, GUIs, etc.). Clients adhere to the Fermat Client Protocol.
```

Comment:
@daveo90 I am not sure of this change..because the components are NOT a different thing than the framework..but they BUILD ALLTOGETHER the framweork, or what the framework is, i.e. the set of functionalities provided by all the components as a WHOLE! please review this! Maybe: ..which is the result of the functionalities of all its components,....